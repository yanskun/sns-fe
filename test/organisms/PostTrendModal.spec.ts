import { createLocalVue, shallowMount } from '@vue/test-utils'
import VueCompositionApi from '@vue/composition-api'
import Vuetify from 'vuetify'
import Component from '~/components/organisms/PostTrendModal.vue'

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
localVue.use(Vuetify)

describe('PostTrendModal', () => {
  test('snapshot', () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        options: [
          { text: 'front', value: 'front' },
          { text: 'server', value: 'server' },
        ],
        ogp: {
          href: 'https://github.com/',
          src: 'https://github.githubassets.com/images/modules/open_graph/github-logo.png',
          title: 'GitHub',
          description:
            'GitHub is where people build software. More than 65 million people use GitHub to discover, fork, and contribute to over 200 million projects.',
        },
      },
      localVue,
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
