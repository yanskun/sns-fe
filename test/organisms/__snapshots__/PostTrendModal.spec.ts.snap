// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostTrendModal snapshot 1`] = `
<v-row-stub
  justify="center"
  tag="div"
>
  <v-dialog-stub
    closedelay="0"
    contentclass=""
    maxwidth="none"
    opendelay="0"
    origin="center center"
    retainfocus="true"
    transition="dialog-transition"
    width="auto"
  >
    <v-card-stub
      loaderheight="4"
      tag="div"
    >
      <v-card-title-stub>
        <p
          class="text-h5"
        >
          Trend
        </p>
      </v-card-title-stub>
      <v-card-text-stub>
        <v-container-stub
          tag="div"
        >
          <v-row-stub
            tag="div"
          >
            <v-col-stub
              cols="12"
              tag="div"
            >
              <v-select-stub
                appendicon="$dropdown"
                backgroundcolor=""
                chips="true"
                clearicon="$clear"
                deletablechips="true"
                errorcount="1"
                errormessages=""
                itemcolor="primary"
                itemdisabled="disabled"
                items="[object Object],[object Object]"
                itemtext="text"
                itemvalue="value"
                label="カテゴリー"
                loaderheight="2"
                menuprops="[object Object]"
                messages=""
                multiple="true"
                nodatatext="$vuetify.noDataText"
                rules=""
                successmessages=""
                type="text"
                value=""
                valuecomparator="[Function]"
              />
            </v-col-stub>
            <v-col-stub
              cols="6"
              tag="div"
            >
              <v-text-field-stub
                backgroundcolor=""
                clearable="true"
                clearicon="$clear"
                errorcount="1"
                errormessages=""
                label="URL*"
                loaderheight="2"
                messages=""
                required="required"
                rules=""
                successmessages=""
                type="text"
                value=""
              />
            </v-col-stub>
            <v-col-stub
              class="ogp"
              cols="6"
              tag="div"
            >
              <!---->
            </v-col-stub>
            <v-col-stub
              cols="12"
              tag="div"
            >
              <v-textarea-stub
                backgroundcolor=""
                clearable="true"
                clearicon="$clear"
                errorcount="1"
                errormessages=""
                label="コメントを入力"
                loaderheight="2"
                messages=""
                name="input-comment"
                rowheight="24"
                rows="5"
                rules=""
                solo="true"
                successmessages=""
                type="text"
                value=""
              />
            </v-col-stub>
          </v-row-stub>
        </v-container-stub>
      </v-card-text-stub>
      <v-card-actions-stub
        class="btn-wrapper"
      >
        <v-spacer-stub>
          <v-btn-stub
            activeclass=""
            color="green darken-1"
            tag="button"
            text="true"
            type="button"
          >
            キャンセル
          </v-btn-stub>
          <v-btn-stub
            activeclass=""
            color="green darken-1"
            tag="button"
            text="true"
            type="button"
          >
            投稿
          </v-btn-stub>
        </v-spacer-stub>
      </v-card-actions-stub>
    </v-card-stub>
  </v-dialog-stub>
</v-row-stub>
`;
